### PR TITLE
239 rebranding

### LINKED ISSUE #239: [IMPROVEMENT] - Weekend-Fly rebranding process
**Issue Description**
Once we agree on the new branding, we need to apply it across all the project, including `mailers`.

**Desired Behavior**
After discussion with Maria:
- orange could be secondary color
- warm colors (accent) for small details
- footer in blue
- Homepage sections between pure white and maybe beige #F8F4EC
- Montserrat should be used as second font
- pages mainly in white

**Acceptance Criteria**
...

**Additional Considerations**
Redesign some screens having radio buttons or selectors (trip_request, preferences, ..)
https://kitt.lewagon.com/knowledge/tutorials/css_advanced_form_checkboxes

**Resources**
...

**Impact**
...

**Related Issues**
...

### PR DESCRIPTION
**Pull request by christinasugiono**
Screenshot 2024-04-30 at 11 37 47
image
Screenshot 2024-04-30 at 11 41 57

closes #239 

NEXT STEP:
- New POI icons from Maria ✨

### CODE REVIEW

File: app/assets/stylesheets/pages/_console.scss
Comment (alexstan67) [inline app/assets/stylesheets/pages/_console.scss:1]: Console.scss should be removed as not used anymore. Replaced by blazer.

File: app/assets/stylesheets/pages/_index.scss [hunk 1]
```diff
@@ -2,3 +2,4 @@
 @import "console";
```
Comment (alexstan67) [inline app/assets/stylesheets/pages/_index.scss:2]: Delete this line

File: app/models/trip_request.rb
Comment (alexstan67) [inline app/models/trip_request.rb:1]: There is a translation issue in french

File: app/views/airports/show.html.erb [hunk 2]
```diff
@@ -278,31 +278,35 @@
...
-              </div>
-            <% end %>
-          </div>
+            </div>
+          <% end %>
         </div>
-    <% end %>
+        <%= link_to "See #{@upcoming_events.count - 4} more +", events_path, class: "link-thin link-dark mt-20" if @upcoming_events.count > 4 %>
+      <% else %>
+        <p><%= t('airport.show.no_events') %></p>
```
Comment (alexstan67) [inline app/views/airports/show.html.erb:303]: I think this is the missing translation

File: app/views/trip_requests/_form.html.erb [hunk 3]
```diff
@@ -47,160 +46,195 @@
...
-            <%= f.label :fuel_station_100ll %>
-          </div>
-          <div class="d-flex flex-row align-items-center mb-5">
-            <%= f.check_box :fuel_station_91ul %>
-            <%= image_tag(FuelStation.inventory[:fuel_types][:fuel_avgas_91ul][:icon], size: "24x24", alt:"fuel_91ul", class: "mr-10") %>
-            <%= f.label :fuel_station_91ul %>
+      </div>
+
+      <div class="mb-5">
+        <label class="form-label text-small text-uppercase"><%= t('trip_request.categories.transportations').upcase %></label>
```
Comment (alexstan67) [inline app/views/trip_requests/_form.html.erb:166]: The translations are missing for fr I think

**Review by alexstan67 — CHANGES_REQUESTED** [review]
It is a HUGE work, well done!
I found small problems mainly on translations, that means that our tests are not efficient enough, I'll work on it (already begun).
- I also saw a problem on `TripRequest#new` when you loading animation appears: The airport search box doesn't seem to have the correct opacity.
- On the `TripSuggestions#index`, the flight duration are not displayed properlly. It works on production. If flight time > 60 minutes, we display in hours and minutes.
- Still on same page, when you take a point of interest on the map, the header's size is way too big.
- On `Airports#show`, personnaly I don't like the different colors depending on the airport_type... I would stick to one color... Or we only stick to accent colors, but no primary for example.
Again, well done for this work! :clap:

File: app/views/trip_requests/_form.html.erb (hunk 3 above)
Comment (christinasugiono) [reply app/views/trip_requests/_form.html.erb:166]: For the `trip_suggestions#index` display time, is this what you meant? 
image

**Review by alexstan67 — APPROVED** [review]

**Pull request merged**